stitching/cli/stitch.py

```diff
@@ -251,7 +251,12 @@
     help="The default is 'result.jpg'",
     type=str,
 )
-
+parser.add_argument(
+    "--mask_names",
+    nargs='*',
+    action="store",
+    type=str,
+)
```

Comment:
100% agree

> So this will be images? 

Yes currently I'm using black/white images + `cv.threshold`, this is not the most effective way to store it, but it sounds to me the easiest way to use it for a newbie.

Another idea would be to store the mask directly in the images, for example as alpha channel, even if it sounds cool, i feel this is more complex to use.